tests/providers/openlineage/plugins/test_listener.py

```diff
@@ -269,14 +268,21 @@ def mock_task_id(dag_id, task_id, execution_date, try_number):
     mocked_adapter.build_dag_run_id.side_effect = lambda x, y: f"{x}.{y}"
     mock_disabled.return_value = False
 
-    listener.on_task_instance_failed(None, task_instance, None)
+    err = ValueError("test")
+    on_task_failed_listener_kwargs = {"error": err} if AIRFLOW_V_2_10_PLUS else {}
+    expected_err_kwargs = {"error": err if AIRFLOW_V_2_10_PLUS else None}
```

Comment:
Can we simulate `AIRFLOW_V_2_10_PLUS` by mock.patch or otherwise rather than relying on environment?

Reply:
Not sure i understand how you'd like to do it ? I saw those variables in `tests/test_utils/compat.py` and i thought that it's designed for that very purpose.

Reply:
Just mock patch decorator on method?

Reply:
Ok, understood why this is not feasible.